test/lib/call-me-back-maybe.js

```diff
@@ -0,0 +1,34 @@
+// A script, used by test/side-effects.js, to test that the client.flush
```

Comment:
[2012 called](https://www.google.com/search?client=firefox-b-1-d&q=call+me+maybe) ;)

Reply:
Jepsen is canon. :)